src/frontends/lean/widget.cpp

```diff
@@ -216,13 +216,13 @@ void reconcile_children(std::vector<vdom> & new_elements, std::vector<vdom> cons
 }
 
 
-void render_event(std::string const & name, vm_obj const & handler, std::map<std::string, unsigned> & events, std::map<unsigned, ts_vm_obj> & handlers) {
-    unsigned handler_id = g_fresh_handler_id.fetch_add(1);
```

Comment:
The global `g_fresh_handler_id` variable should also be removed.

Reply:
oops